pkg/common/cns-lib/volume/listview.go

```diff
@@ -324,13 +324,22 @@ func (l *ListViewImpl) listenToTaskUpdates() {
 		l.mu.Lock()
 		log.Infof("acquired lock before re-creating listview")
 		if recreateView {
+			if l.listView != nil {
+				destroyListviewErr := l.listView.Destroy(l.ctx)
+				if destroyListviewErr != nil {
+					log.Errorf("failed to destroy listview object. err: %v", destroyListviewErr)
```

Comment:
can you update the comment saying that you will be ignoring the error and re-creating the list view again?